src/prebid.js

```diff
@@ -757,6 +761,18 @@ $$PREBID_GLOBAL$$.setS2SConfig = function(options) {
   adaptermanager.setS2SConfig(config);
 };
 
+/**
+ * Get Prebid config options
+ * @param {object} options
+ */
+$$PREBID_GLOBAL$$.getConfig = config.getConfig;
+
+/**
+ * Set Prebid config options
+ * @param {object} options
```

Comment:
More specific JSDocs would be super helpful here.

I understand we should probably merge this PR to unblock people, though... so can we make it a priority shortly after? It improves the developer experience a ton.